DQMOffline/Trigger/plugins/BPHMonitor.cc

```diff
@@ -0,0 +1,342 @@
+#include "DQMOffline/Trigger/plugins/BPHMonitor.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+
+double MAX_PHI = 3.2, MAX_ETA = 2.6, MAX_PT = 200, MAX_D0 = 100, MAX_Z0 = 150;
```

Comment:
in addition, I would suggest to have histograms w/ variable binning for both the pT and eta
probably the same applies to d0 and dz